buildspec.test.yml

```diff
@@ -3,16 +3,18 @@ version: 0.2
 phases:
   install:
     commands:
-      - echo Setting local Python versions
+      - echo -e "\nSetting local Python versions"
```

Comment:
Dropped some line break around to make it easier to see where the different parts begin and end in the nasty lofi CodeBuild output.